css/variables.css

```diff
@@ -0,0 +1,47 @@
+html {
+  /** Colors **/
+  --color-red: #ec2227;
+  --color-orange: #f79421;
+  --color-black: #000;
+  --color-grey: #666;
+
+  /** Spacer **/
+  --spacer-2xs: 0.25rem; /* 4px */
+  --spacer-xs: 0.5rem;   /* 8px */
+  --spacer-sm: 1rem;     /* 16px */
+  --spacer-base: 1.5rem; /* 24px */
+  --spacer-lg: 2rem;     /* 32px */
+  --spacer-xl: 2.5rem;   /* 40px */
+  --spacer-2xl: 5rem;    /* 80px */
+  --spacer-3xl: 10rem;   /* 160px */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+body {
+  /** Typography **/
+  --font-family--heading: "Montserrat", sans-serif;
+  --font-family--subheading: "Open Sans", sans-serif;
+
+  --font-weight--light: 300;
+  --font-weight--normal: 400;
+  --font-weight--medium: 500;
+  --font-weight--semibold: 600;
+  --font-weight--bold: 700;
+
+  --font-size--xs: 0.75rem;  /* 12px */
+  --font-size--sm: 0.875rem; /* 14px */
+  --font-size--base: 1rem;   /* 16px */
+  --font-size--lg: 1.125rem; /* 18px */
+
+  --h1-font-size: 2.625rem;               /* 42px */
```

Comment:
customize font sizes to match current website. for example, H1 is currently 56px

Reply:
Done

Reply:
There are so many font sizes like 11px, 17px 41px etc given to tags. So I have created variables according to approximate values of font sizes used.